src/main/java/soot/manager/CommandManager.java

```diff
@@ -0,0 +1,118 @@
+package soot.manager;
+
+import soot.EmptyTaskException;
+import soot.task.Deadline;
+import soot.task.Event;
+import soot.task.Task;
+import soot.task.Todo;
+
+public class CommandManager {
+    public static int listCounter;
+    public static final int LIST_SIZE = 100;
+    public static Task[] taskList; //public static?
+
+    public CommandManager() {
+        taskList = new Task[LIST_SIZE];
+        listCounter = 0;
+    }
+
+    public static String getCommand(String input) throws EmptyTaskException {
+        if (input.equals("bye") || input.equals("list")) {
+            return input;
+        }
+
+        int spaceIndex = input.indexOf(" ");
+        if (spaceIndex == -1) {
+            throw new EmptyTaskException();
+        }
+        return input.substring(0, spaceIndex);
+    }
+    public static boolean isInputBye(String input) {
+        String inputCommand = input;
+        try {
+            inputCommand = getCommand(input);
+            String inputTask;
+            switch (inputCommand) {
+            case "bye":
+                return true;
+            case "list":
+                System.out.println("tasks to be done:");
+                if (listCounter == 0) {
+                    System.out.println("  >> nothing so far :)");
+                    drawLine();
+                    break;
+                }
+                for (int i = 0; i < listCounter; i++)
+                    taskList[i].printTask();
+                drawLine();
+                break;
+            case "done":
+                inputTask = input.substring(5);
+                int listIndex = Integer.parseInt(inputTask) - 1;
```

Comment:
Instead of repeating this for every case, consider writing a method to get desired index and invoke the method here.